script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -175,20 +175,20 @@ Those idiots have gone in and tried to turn it into some kind of scam. I don't k
 <Page48>
 "Basically, I don't wanna get hurt, right? If they keep being so brazen, the police will step in soon. I'll just stay indoors for a while until they're caught."
 <Page49>
-The urban legend of the north gate... Unfortunately, I am not a local, so it is unfamiliar to me.
+The urban legend of the north gate... Unfortunately, I am not a local, so it's unfamiliar to me.
 After today, it won't be a strange thing to me anymore... I think.
 <Page50>
 "Hey, what's with the lack of response?
 It must be that, right? Did you collapse from anemia this morning?"
 <Page51>
-"No, I'm fine this morning.
+"No, I'm fine.
 It's nice of you to worry about me, but my body wouldn't be able to take it if I'm that anemic all the time."
 "Yeah, I guess you're right. If you say you're okay, then you must be okay."
 <Page52>
 "Come on, class is about to start. Get back to your seat and go to sleep."
 "Right. ...Oh, yeah. For today's lunch, I'll be in the cafeteria. I've invited a special guest to surprise you, so look forward to it."
 <Page53>
-Laughing as if he is planning something, Arihiko goes back to his seat.
+Laughing as if he were planning something, Arihiko goes back to his seat. //Marked to recheck
```

Comment:
> きしし、と何か企んでいそうな笑い声をあげて、有彦は自分の席に戻っていった。

Reply:
Arihiko returned to his seat laughing as if he were plotting something